.vscode/launch.json

```diff
@@ -0,0 +1,18 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "type": "node",
+            "request": "launch",
+            "name": "Custom Functions Web Debugging",
+            "program": "${workspaceFolder}/node_modules/office-addin-node-debugger/lib/debugger.js",
+            "args": ["dev-server"],
```

Comment:
>"args": ["dev-server"], [](start = 12, length = 23)

What does this argument do? So far, the debugger only accepts **-p** and **-h** for defining the port and host respectively.
Perhaps that argument was from an old version pointing to the "Start-debugging" package.

Reply:
probably so, will remove